test/remote/gateways/remote_payu_latam_test.rb

```diff
@@ -0,0 +1,213 @@
+require "test_helper"
+
+class RemotePayuLatamTest < Test::Unit::TestCase
+  def setup
+    @gateway = PayuLatamGateway.new(fixtures(:payu_latam))
+
+    @amount = 10000
+    @credit_card = credit_card("4097440000000004", verification_value: "444", first_name: "APPROVED", last_name: "")
+    @declined_card = credit_card("4097440000000004", verification_value: "333", first_name: "REJECTED", last_name: "")
+    @master_card = credit_card('5105105105105100', :verification_value => '999', first_name: "APPROVED", last_name: "", :brand => 'master')
+
+    @options = @options_argentina = {
+      currency: "ARS",
+      order_id: generate_unique_id,
+      email: "payu_latam@example.com",
+      description: "Argentina Transaction",
+      ip: "127.0.0.1",
+      installments_number: 1,
+      billing_address: address(address1: "Viamonte",
+                       address2: "1366",
+                       city: "Plata",
+                       state: "Buenos Aires",
+                       country: "AR",
+                       zip: "64000",
+                       phone: "7563126")
+    }
+
+    @options_brazil = {
```

Comment:
As there was lot of back n forth while testing this gateway, I had at various times tried out each different `account_id` that maps to a different country as you rightly guessed. In the final commit, I just left them all in.

Pattern wise, there is Brazil and the rest, so this can be trimmed down to just those two.